interface/main/about_page.php

```diff
@@ -128,10 +128,23 @@
     </div>
 
 
-    <div class="product-registration-modal" style="display: none">
-        <p class="context"><?php echo xlt("Register your installation with OEMR to receive important notifications, such as security fixes and new release announcements."); ?></p>
-        <input placeholder="<?php echo xla('email'); ?>" type="email" class="email w-100" style="color: var(--black)" />
-        <p class="message" style="font-style: italic"></p>
+    <div class="product-registration-modal modal fade">
+        <div class="modal-dialog modal-dialog-centered">
+            <div class="modal-content">
+                <div class="modal-header"></div>
+                <div class="modal-body">
+                    <p class="context"><?php echo xlt("Register your installation with OEMR to receive important notifications, such as security fixes and new release announcements."); ?></p>
+                    <input placeholder="<?php echo xlt('email'); ?>" type="email" class="email w-100 text-body" />
+                    <p class="message font-italic"></p>
+                </div>
+                <div class="modal-footer">
+                    <div class="btn-group" role="group">
+                        <button class="btn btn-primary submit">Submit</button>
+                        <button class="btn btn-danger noThanks">No Thanks</button>
```

Comment:
1. Translate/escape
2. Avoid using caps in class since not sure what impact that might have
```
                                    <button class="btn btn-primary submit"><?php echo xlt("Submit"); ?></button>
                                    <button class="btn btn-danger nothanks"><?php echo xlt("No Thanks"); ?></button>
```

Reply:
Additionally, rather than use a class, would identify the button with a id and use this instead to register the click in product_registration_controller.js